src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -82,11 +85,12 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
 
   const onSubmit = async (data: RegisterFormData) => {
```

Comment:
Is this onSubmit function the same between both forms? If so, can we extract it to a common function as well?

Reply:
Good catch. There are a few key differences that mean it'd have to be progressively DRYed up:

* `signUp`/`signIn`
* append to `codes_used` (in `LogInForm`) vs set it for the first time (in `RegisterForm`)

OTOH the stuff that's the same includes

* Validation
* Post login steps

Would you be OK to hold off and DRY this up after I've removed the burn-related stuff from it?

Reply:
Happy for it to be a future cleanup thing. I don't think it's worth blocking the other improvements here for it.